File: src/main/resources/static/js/service/loggerService.js
app.service('loggerService',function ($http) {
    this.findAll = function () {
        return $http.get('../logger/findAll');
    };

    this.search = function (pageNo,pageSize,searchLogger) {
        return $http.post("../logger/search?pageNo="+pageNo+"&pageSize="+pageSize,searchLogger);
    };
});

File: src/main/java/com/shengxuan/speed/service/LoggerService.java
package com.shengxuan.speed.service;

import com.shengxuan.speed.entity.Logger;
import com.shengxuan.speed.entity.pojo.PageResult;

import java.util.List;

public interface LoggerService {
    List<Logger> findAll();

    /**
     * 分页条件查询
     * @param logger
     * @param pageNo
     * @param pageSize
     * @return
     */
    PageResult findPage(Logger logger, int pageNo, int pageSize);

}
